src/tests/ftest/security/pool_connect_init.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2017-2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from __future__ import print_function
+import os
+import traceback
+from apricot import TestWithServers
+from daos_api import DaosPool, DaosApiError
+
+class PoolSecurityTest(TestWithServers):
+    """
+    DAOS security: initial security tests after pool creation
+    :avocado: recursive
+    """
+    def setUp(self):
+        """
+        set up method: create 3 pools with different uid and gid
+                       for security test.
+            pool1: with owner uid, owner-group root-id.
+            pool2: with owner rooter-id, owner-group gid.
+            pool3: owner and owner-group with both root-id.
+        """
+        super(PoolSecurityTest, self).setUp()
+        # get parameters from yaml file
+        mode = self.params.get("mode", '/run/poolparams/createmode/', 511)
+        setid = self.params.get("setname", '/run/poolparams/createset/',
+                                "daos_server")
+        size = self.params.get("size", '/run/poolparams/createsize/',
+                               107374182)
+        root_id = 0
+        user_uid = os.geteuid()
+        user_gid = os.getegid()
```

Comment:
Would it make sense to save this as self.owner_uid and self.owner_gid, and then reusing below?